data/store/mongo/mongo_data.go

```diff
@@ -40,20 +40,24 @@ func (d *DataRepository) ListUserDataSets(ctx context.Context, userID string, fi
 }
 
 func (d *DataRepository) GetDataSet(ctx context.Context, id string) (*data.DataSet, error) {
-	// Try reading from both new and old collections that hold dataSets, starting with the new one.
-	// Can read only from the new deviceDataSets collection via DataSetRepository when migration completed.
-	dataSet, err := d.DataSetRepository.GetDataSet(ctx, id)
+	return d.DataSetRepository.GetDataSet(ctx, id)
+}
+
+func (d *DataRepository) GetDataSetByID(ctx context.Context, dataSetID string) (*upload.Upload, error) {
+	return d.DataSetRepository.GetDataSetByID(ctx, dataSetID)
+}
+
+func (d *DataRepository) IsDataSetAutomated(ctx context.Context, dataSetID string) (bool, error) {
```

Comment:
I don't think we should introduce a new term `automated` to avoid confusion. We are referring to those datasets as `continuous`.

Reply:
The definition of "automated" contained many more conditions originally which was under debate, so it used to be distinct, but now I agree this is silly.

I have added `IsContinuous()` to `Upload` type.